opm/simulators/wells/GasLiftSingleWell_impl.hpp

```diff
@@ -143,7 +143,7 @@ computeBhpAtThpLimit_(Scalar alq, bool debug_output) const
         this->simulator_,
         this->summary_state_,
         alq,
-        this->deferred_logger_);
+        this->deferred_logger_, /*iterate_if_no_solution */false);
```

Comment:
Very tiny nitpicking here,  a space before `false` and this arguments goes to a separate line to be consistent with other arguments.